blueprints/plugin_testing/create_admin_resources_test-blueprint.yaml

```diff
@@ -0,0 +1,141 @@
+#Blueprint used for testing creation, deletion and other operations for these node_types:
+#
+#* cloudify.openstack.nodes.Flavor
+#* cloudify.openstack.nodes.Image
+#* cloudify.openstack.nodes.Project
+#* cloudify.openstack.nodes.User
+#* cloudify.openstack.nodes.HostAggregate
+#
+
+tosca_definitions_version: cloudify_dsl_1_3
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/4.2/types.yaml
+  - https://raw.githubusercontent.com/cloudify-cosmo/cloudify-openstack-plugin/master/plugin.yaml
+
+inputs:
+  openstack_username:
+    type: string
+    default: { get_secret: keystone_username }
+
+  openstack_password:
+    type: string
+    default: { get_secret: keystone_password }
+
+  openstack_auth_url:
+    type: string
+    default: { get_secret: keystone_url }
+
+  openstack_region:
+    type: string
+    default: { get_secret: keystone_region }
+
+  openstack_admin_tenant_name: # should admin_tenant name be required to provision new tenet / user ?
+    type: string
+    default: 'admin'
+
+node_templates:
+  test_user:
+    type: cloudify.openstack.nodes.User
+    properties:
+      user:
+        name: 'openstack_plugin_test_user'
+      use_external_resource: false
+      resource_id: 'openstack_plugin_test_user'
+      openstack_config:
```

Comment:
@kbijakowski I think we can use dsl_definitions here 
```
dsl_definitions:
   openstack_config: &openstack_config
      username: { get_input: openstack_username }
      password: { get_input: openstack_password }
      tenant_name: { get_input: openstack_project_name }
      auth_url: { get_input: openstack_auth_url }
      region: { get_input: openstack_region }
```

and then use it below in all nodes

Reply:
DONE (only for places where admin tenant config is used)